### DISCUSSION TITLE
[Feature] section to see the latest added photos and videos

### DISCUSSION THREAD

**Discussion by dvelayos** [immich-app/immich#10455] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

the photos section order the contents by the date time the photo/ video has been taken, but if you add some old photos / videos to immich, there is no way to see easily the recent additions (you have to navigate to the year/month on which the photos has been taken)

it would be great if we could get a new section called "recently added" or something like this below the "photos" option, for example, or inside the "explore" section (adding a new section below "Places")

### Platform

- [x] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#8716

**Comment by dvelayos** [comment]
swear to god that i searched on features and i didn't see the feature reported, sorry
